.storybook-css/addons.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies, import/extensions */
```

Comment:
why are these deps extraneous?

Reply:
They're never used directly!

Reply:
(but they are necessary)

Reply:
They're imported here; "extraneous" means they're not in package.json. If they're necessary, why can't they be in package.json?

Reply:
So the thing is... they come with `@storybook/react` but they have namespacing that makes eslint think they're separate packages. idk man, i blame storybook for this one, but this is what you have to do to get them to work.

Reply:
aren't they separate packages? https://www.npmjs.com/package/@storybook/addon-actions and https://www.npmjs.com/package/@storybook/addon-links

Reply:
I don't believe they were when I first put together this addons.js file/updated storybook in this package. It's possible this has changed in newer versions.

This is literally just a copy of what's in `.storybook` which is the default config.